applications/MeshMovingApplication/python_scripts/ale_fluid_solver.py

```diff
@@ -16,51 +16,61 @@ def CreateSolver(model, solver_settings, parallelism):
 
 class ALEFluidSolver(PythonSolver):
     def __init__(self, model, solver_settings, parallelism):
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "solver_type"                 : "ale_fluid",
+            "fluid_solver_settings"       : { },
+            "mesh_motion_solver_settings" : { },
+            "echo_level"                  : 0,
+            "ale_boundary_parts"          : [ ]
+        }""")
+
+        # cannot recursively validate bcs validation of fluid- and
```

Comment:
I was confused by this comment because I thought `bcs` stood for boundary conditions :confused:

Reply:
fixed